go-controller/pkg/ovn/ovn.go

```diff
@@ -208,8 +221,28 @@ const (
 
 	// SCTP is the constant string for the string "SCTP"
 	SCTP = "SCTP"
+
+	// IPv4FullMask is the maximum prefix mask for an IPv4 address
+	IPv4FullMask = "/32"
```

Comment:
@rcarrillocruz can remove these now that they are in the function

Reply:
oops, ctrl-x fail, done